component/componenterror/errors.go

```diff
@@ -37,27 +35,5 @@ var (
 
 // CombineErrors converts a list of errors into one error.
 func CombineErrors(errs []error) error {
```

Comment:
I guess it's better to deprecate it in the next PR that will change this function to use `consumererror.CombineErrors(errs)` since we decided that it's better to leave it as-is in the current PR.